docs/user-guide/configuring-at-tls-for-zowe-server.md

```diff
@@ -32,15 +32,15 @@ Follow these steps to configure Zowe to support AT-TLS:
 
 ```yaml
 zowe:
-    network:
-        # For inbound traffic rules:
-        server:
-            tls:
-                attls: true
-        # If outbound traffic rules will be configured:
-        client:
-          tls:
-            attls: true
+  network:
+    # For inbound traffic rules:
+    server:
+      tls:
+        attls: true
+    # If outbound traffic rules will be configured:
+    client:
+      tls:
+        attls: true
```

Comment:
fixed this, it was a bug. it appeared to have more spaces then you'd want for yaml